tests/components/onewire/__init__.py

```diff
@@ -89,6 +89,9 @@ def check_entities(
         )
         for attr in FIXED_ATTRIBUTES:
             assert state.attributes.get(attr) == expected_entity.get(attr)
+        assert registry_entry.entity_category == expected_entity.get(
+            ATTR_ENTITY_CATEGORY
+        )
```

Comment:
This is no longer required, already present on line 81:
https://github.com/home-assistant/core/blob/4144699814602e0c116bf801c2f8f26ba25edca6/tests/components/onewire/__init__.py#L81-L83

Reply:
```suggestion
```